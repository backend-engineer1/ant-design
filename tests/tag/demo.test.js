import demoTest from '../shared/demoTest';

demoTest('tag');
