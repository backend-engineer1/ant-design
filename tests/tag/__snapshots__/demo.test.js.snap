exports[`test renders ./components/tag/demo/basic.md correctly 1`] = `
<div>
  <div
    class="ant-tag"
    data-show="true">
    <span
      class="ant-tag-text">
      Tag 1
    </span>
  </div>
  <div
    class="ant-tag"
    data-show="true">
    <span
      class="ant-tag-text">
      Tag 2
    </span>
    <i
      class="anticon anticon-cross" />
  </div>
  <div
    class="ant-tag"
    data-show="true">
    <span
      class="ant-tag-text">
      Tag 3
    </span>
    <i
      class="anticon anticon-cross" />
  </div>
  <div
    class="ant-tag"
    data-show="true">
    <span
      class="ant-tag-text">
      <a
        href="https://github.com/ant-design/ant-design/issues/1862">
        Link
      </a>
    </span>
    <i
      class="anticon anticon-cross" />
  </div>
  <div
    class="ant-tag"
    data-show="true">
    <span
      class="ant-tag-text">
      Prevent Default
    </span>
    <i
      class="anticon anticon-cross" />
  </div>
</div>
`;

exports[`test renders ./components/tag/demo/checkable.md correctly 1`] = `
<div>
  <div
    class="ant-tag ant-tag-checkable">
    Unchecked
  </div>
  <div
    class="ant-tag ant-tag-checkable ant-tag-checkable-checked">
    Checked
  </div>
  <div
    class="ant-tag ant-tag-checkable">
    Uncontrolled
  </div>
</div>
`;

exports[`test renders ./components/tag/demo/colorful.md correctly 1`] = `
<div>
  <div
    class="ant-tag ant-tag-#f50 ant-tag-has-color"
    data-show="true"
    style="background-color:#f50;">
    <span
      class="ant-tag-text">
      #f50
    </span>
  </div>
  <div
    class="ant-tag ant-tag-#87d068 ant-tag-has-color"
    data-show="true"
    style="background-color:#87d068;">
    <span
      class="ant-tag-text">
      #87d068
    </span>
  </div>
  <div
    class="ant-tag ant-tag-#108ee9 ant-tag-has-color"
    data-show="true"
    style="background-color:#108ee9;">
    <span
      class="ant-tag-text">
      #108ee9
    </span>
  </div>
</div>
`;

exports[`test renders ./components/tag/demo/control.md correctly 1`] = `
<div>
  <div
    class="ant-tag"
    data-show="true">
    <span
      class="ant-tag-text">
      Unremovable
    </span>
  </div>
  <div
    class="ant-tag"
    data-show="true">
    <span
      class="ant-tag-text">
      Tag 2
    </span>
    <i
      class="anticon anticon-cross" />
  </div>
  <div
    class="ant-tag"
    data-show="true">
    <span
      class="ant-tag-text">
      Tag 3
    </span>
    <i
      class="anticon anticon-cross" />
  </div>
  <button
    class="ant-btn ant-btn-dashed ant-btn-sm"
    type="button">
    <span>
      + New Tag
    </span>
  </button>
</div>
`;

exports[`test renders ./components/tag/demo/hot-tags.md correctly 1`] = `
<div>
  <strong>
    Hots: 
  </strong>
  <div
    class="ant-tag ant-tag-checkable">
    Movie
  </div>
  <div
    class="ant-tag ant-tag-checkable">
    Books
  </div>
  <div
    class="ant-tag ant-tag-checkable">
    Music
  </div>
</div>
`;
